content/15-tools/02-release-notes.mdx

```diff
@@ -11,17 +11,17 @@ Cardano node configuration details (for mainnet and testnets) are available on t
 
 ## Latest Cardano component releases
 
-Name | Version | Release Date
+Name | Version | Release date
 -|-|-
-[Cardano Node](https://github.com/input-output-hk/cardano-node) | [8.1.2](https://github.com/input-output-hk/cardano-node/releases/tag/8.1.2) | 24 July, 2023
+[Cardano node](https://github.com/input-output-hk/cardano-node) | [8.7.2](https://github.com/input-output-hk/cardano-node/releases/tag/8.7.2) | 8 December, 2023
 [Cardano RTView](https://github.com/input-output-hk/cardano-rt-view) | [0.3.0](https://github.com/input-output-hk/cardano-rt-view/releases/tag/0.3.0) | 15 January, 2021
```

Comment:
you mother fucker dont cheat in stock market trading undwerstand or i will shut all githup down mother fucker you tell true and lie but dont tell cheat